tests/e2e_tests/cypress/e2e/basic-test/commons.js

```diff
@@ -0,0 +1,88 @@
+const expectedPlace1 = {
+  title: "Grunwaldzki",
+  subtitle: "big bridge",
+  categories: [
+    ["type_of_place", "big bridge"],
+    ["accessible_by", "pedestrians, cars"]
+  ],
+  CTA: {
+    "type": "CTA",
+    "value": "https://www.example.com",
+    "displayValue": "Visit example.org!"
+  }
+};
+const expectedPlace2 = {
+  title: "Zwierzyniecka",
+  subtitle: "small bridge",
+  categories: [
+    ["type_of_place", "small bridge"],
+    ["accessible_by", "bikes, pedestrians"]
+  ]
+};
+
+export const expectedPlaces = [expectedPlace1, expectedPlace2];
+
+const zoomInTimes = 1;
+
+export function zoomInMap() {
+  for (let i = 0; i < zoomInTimes; i++) {
+    cy.get('.leaflet-marker-icon').first().click();
+    cy.wait(500);
+  }
+}
```

Comment:
Thats kind of silly to iterate when you give global limit to one.

Also... this ain't no zoom at all :P its rather: `clickSomeMarkerXtimes`